tests/Basic/BasicAuthTest.php

```diff
@@ -20,12 +20,13 @@ public function testSetBasicAuth(string $user, string $pass, string $pageText):
         $this->assertStringContainsString($pageText, $session->getPage()->getContent());
     }
 
+    /**
+     * @return iterable<string, array{string, string, string}>
```

Comment:
If the data type is PhpStan-specific, please use a different annotation for that as @stof mentioned in some recent PRs to different project repos.

P.S.
This applies to other places in this PR, where DocBlock uses PhpStan-specific stuff.

Reply:
I don't believe that is PHPStan-specific. The other point was about type aliases, which we can only define with `@phpstan-type TXx`.

The difference is that some tools might not parse vendor-specific annotations will then see the TXx as a *missing types*.

The type syntax complexity is a different problem which I believe we should not try to solve (e.g. some tools might also not support `Xx[]` - I don't think it makes sense to cater for them).

Reply:
OK. Then we'll keep it.